docs/source/tutorial.rst

```diff
@@ -376,13 +376,36 @@ in MLflow saved the model as an artifact within the run.
           ``UnicodeDecodeError: 'ascii' codec can't decode byte 0x9f in position 1: ordinal not in range(128)``
           or ``raise ValueError, "unsupported pickle protocol: %d"``.
 
-      To serve a prediction, run:
+      Once you have deployed the server, you can pass it some sample data and see the
```

Comment:
Duplicating this documentation is weird, as is the place it's located. It seems like neither quickstart nor tutorial should probably go into this much information about the serving formats. Can we put this documentation in some serving-specific location, and have both link to that?